source/mus_config_module.f90

```diff
@@ -194,6 +202,11 @@ subroutine mus_load_config( scheme, solverData, geometry, params, adapt )
       &                   geometry   = geometry,                      &
       &                   conf       = params%general%solver%conf(1), &
       &                   params     = params                         )
+    
+    ! For coupled LBM-DEM simulations with solid particles
+    ! load data for particles from 'particles' table in musubi.lua, if present
+    call mus_load_particlekind( particleGroup = particleGroup,                &
```

Comment:
Data read from the configuration file should in general go into `params`, I think. You might want to split the `particleGroup` into something like `particleHeader` and `particleData`, as it looks like you do some more stuff afterward in `mus_particle_initialize`. The header could maybe go into params and the data into scheme.